pkg/model/model.go

```diff
@@ -41,10 +43,11 @@ type Minecraft struct {
 
 // Java
 type Java struct {
-	Xmx     string `yaml:"xmx"`
-	Xms     string `yaml:"xms"`
-	OpenJDK int    `yaml:"openjdk"`
-	Rcon    Rcon   `yaml:"rcon"`
+	Xmx     string   `yaml:"xmx"`
+	Xms     string   `yaml:"xms"`
+	Options []string `yaml:"options"`
```

Comment:
Should we not add this option to the manifest.json too?

Reply:
Added in new commit, thanks for the catch!